src/0x.ts

```diff
@@ -254,4 +255,45 @@ export class ZeroEx {
         }
         return ecSignature;
     }
+    /**
+     * Returns the ethereum addresses of all available exchange contracts
+     * on the network that the provided web3 instance is connected to
```

Comment:
Returns the ethereum addresses of all available exchange contracts
supported by this library on the network that the supplied web3
provider is connected to